Server/Program.cs

```diff
@@ -30,12 +31,21 @@ public static void Main(string[] args)
                     .PageSize(10)
                     .AddChoices(SerialPort.GetPortNames())
                     .HighlightStyle(new Style(foreground: Color.White, background: Color.Blue))
+                    .AddChoices("Mock Serial Data")
```

Comment:
This should be re-ordered to be more readable:
```suggestion
                    .AddChoices(SerialPort.GetPortNames())
                    .AddChoices("Mock Serial Data")
                    .HighlightStyle(new Style(foreground: Color.White, background: Color.Blue))
```